.tx/config

```diff
@@ -0,0 +1,8 @@
+[main]
+host = https://www.transifex.com
+
+[stanford-openedx-xblocks.image-modal]
```

Comment:
How do we want to group/name these?

It seems like a good idea to give all xblocks a shared Transifex "project" within our team "organization",
separate from the platform strings project. We can then have separate "resources" for each XBlock,
within the parent project.
- Any issues with that?
- Any better ideas?

Assuming, do we care to call the project `stanford-opendx-xblocks`
or simply `xblocks`, since it'll be within the `stanford-online` organization?

So here, either:
- `xblocks.image-modal`
  - https://www.transifex.com/stanford-online/xblocks
- `stanford-online-xblocks.image-moodal`
  - https://www.transifex.com/stanford-online/stanford-openedx-xblocks

Thoughts?

Reply:
@stvstnfrd I like just xblocks, that's also how upstream Transifex has theirs named.